packages/jest-runtime/src/index.ts

```diff
@@ -679,7 +680,7 @@ class Runtime {
 
     const runScript = this._environment.runScript(transformedFile.script);
 
-    if (runScript === null) {
+    if (!hasModuleWrapper(runScript)) {
```

Comment:
revert - it should never be something that not either `null` or the object wrapper.